### PR TITLE
feat(helm): Add instruction to install workflow using the new helm

### PR DESCRIPTION
**Pull request by kmala**

### CODE REVIEW

**Comment by bacongobbler** [comment]
We should probably add a note that this is experimental(?) at this time.

File: src/installing-workflow/index.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
                                               # your cluster
 ```
 
+!!! Experimental
+	Workflow can also be installed now using the [Kubernetes Helm](helm). All the details that are needed for a production deployments
```
Comment (ultimateboy) [inline src/installing-workflow/index.md]: `(helm)` should actually be `[helm]` to correctly link to the url.

File: src/installing-workflow/index.md [hunk 2]
```diff
@@ -75,6 +75,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
...
+  like off-cluster storage, external registry etc., can be configured by passing an optional [values file](valuesfile) which overrides
```
Comment (ultimateboy) [inline src/installing-workflow/index.md]: `(valuesfile)` should be `[valuesfile]` to correctly link to the url.

File: src/quickstart/provider/aws/install-aws.md [hunk 3]
```diff
@@ -52,6 +52,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
                                               # your cluster
 ```
 
+!!! Experimental
+	Workflow can also be installed now using the [Kubernetes Helm](helm). All the details that are needed for a production deployments
```
Comment (ultimateboy) [inline src/quickstart/provider/aws/install-aws.md]: fix link

File: src/quickstart/provider/aws/install-aws.md [hunk 4]
```diff
@@ -52,6 +52,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
...
+  like off-cluster storage, external registry etc., can be configured by passing an optional [values file](valuesfile) which overrides
```
Comment (ultimateboy) [inline src/quickstart/provider/aws/install-aws.md]: fix link

File: src/quickstart/provider/gke/install-gke.md [hunk 5]
```diff
@@ -47,6 +47,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
                                               # your cluster
 ```
 
+!!! Experimental
+	Workflow can also be installed now using the [Kubernetes Helm](helm). All the details that are needed for a production deployments
```
Comment (ultimateboy) [inline src/quickstart/provider/gke/install-gke.md]: fix link

File: src/quickstart/provider/gke/install-gke.md [hunk 6]
```diff
@@ -47,6 +47,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
...
+  like off-cluster storage, external registry etc., can be configured by passing an optional [values file](valuesfile) which overrides
```
Comment (ultimateboy) [inline src/quickstart/provider/gke/install-gke.md]: fix link

File: src/quickstart/provider/vagrant/install-vagrant.md [hunk 7]
```diff
@@ -47,6 +47,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
                                               # your cluster
 ```
 
+!!! Experimental
+	Workflow can also be installed now using the [Kubernetes Helm](helm). All the details that are needed for a production deployments
```
Comment (ultimateboy) [inline src/quickstart/provider/vagrant/install-vagrant.md]: fix link

File: src/quickstart/provider/vagrant/install-vagrant.md [hunk 8]
```diff
@@ -47,6 +47,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
...
+  like off-cluster storage, external registry etc., can be configured by passing an optional [values file](valuesfile) which overrides
```
Comment (ultimateboy) [inline src/quickstart/provider/vagrant/install-vagrant.md]: fix link

Comment (kmala) [inline src/quickstart/provider/vagrant/install-vagrant.md]: fixed all the links.

File: src/installing-workflow/index.md [hunk 9]
```diff
@@ -75,6 +75,15 @@ $ helmc install workflow-v2.7.0               # injects resources into
                                               # your cluster
 ```
 
+!!! Experimental
+	Workflow can also be installed now using the [Kubernetes Helm][helm]. All the details that are needed for a production deployments
+  like off-cluster storage, external registry etc., can be configured by passing an optional [values file][valuesfile] which overrides
+  default values.
+  ```
+  $ helm repo add deis https://charts.deis.com/workflow                                # add the workflow charts repo
+  $ helm install workflow --version=v2.7.0 --namespace=deis -f <optional values file>  # injects resources into your cluster
```
Comment (vdice) [inline src/installing-workflow/index.md]: @kmala I can't get `helm install workflow --version=v2.7.0 --namespace=deis` to work w/o changing it to `helm install deis/workflow ...` -- should the docs be updated to reflect this or am I missing something?

Comment (kmala) [inline src/installing-workflow/index.md]: Its a mistake, i thought i had changed it before.Updated the docs.

**Pull request merged**